packages/block-editor/src/components/url-popover/style.scss

```diff
@@ -58,6 +58,7 @@
 	text-overflow: ellipsis;
 	white-space: nowrap;
 	margin-right: $grid-unit-10;
+	min-width: 150px;
```

Comment:
Do you recall what the value was before?

I think the input field here is 300px. Maybe keep that?
![Screenshot 2024-02-22 at 14 28 01](https://github.com/WordPress/gutenberg/assets/1204802/ee560c3a-1aa9-4237-94ee-e44c000c17ce)

Reply:
> Do you recall what the value was before?

The previous value was 150px and I have restored the same value in this PR.

See: https://github.com/WordPress/gutenberg/pull/57608/files#diff-49bee8a5008288296ba2faca8ec5cb424a7c7d56790d40cf7f28cc6966409662L74